lib/navigation.php

```diff
@@ -50,6 +50,9 @@ function gutenberg_register_navigation_post_type() {
 			'editor',
 			'revisions',
 		),
+		'capabilities'          => array(
+			'create_posts' => 'edit_theme_options',
```

Comment:
Restrict creation of Nav posts to "admin" role level only. Allows us to display a warning immediately upon insertion of Nav block.

Attempting to modify all the permission (as in https://github.com/WordPress/wordpress-develop/pull/2056) will not work because `getEntityRecord` will not return the Navigation Post for viewing.